documentation/docs/identus/identus-cloud-agent/authenticate.md

```diff
@@ -38,12 +38,12 @@ An example of an authentication challenge state:
 
 ```json
 {
-  "self": "https://demo.atalaprism.io/did-authentication/challenges/challenge-id-1234",
+  "self": "https://demo.github.com/hyperledger/identus/did-authentication/challenges/challenge-id-1234",
```

Comment:
Cannot really tell about this link. Is this changed due to a change in another repo (ie identus-cloud-agent)?

Reply:
I am not sure either, I followed the guideline in https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit?pli=1#gid=0

Reply:
You are right, @CryptoKnightIOG, but our landing project `identus` doesn't work as a demo site. 
So, it's better to keep it unchanged.

Reply:
In the replacement table, there is no mentioning of the link `https://demo.atalaprism.io`. I guess you mixed with https://atalaprism.io.